CHANGES.txt

```diff
@@ -42,6 +42,9 @@ Backward Incompatibilities
 Features
 --------
 
+- Python 3.6 compatibility.
```

Comment:
Yeah, if https://github.com/Pylons/pyramid/pull/2840 is merged, I can rebase on master and push, and these 3.6 items should go away.